kaldb/src/main/java/com/slack/kaldb/preprocessor/PreprocessorRateLimiter.java

```diff
@@ -72,10 +72,7 @@ public Predicate<String, Trace.Span> createRateLimiter(
       int bytes = value.toByteArray().length;
       if (serviceName == null || serviceName.isEmpty()) {
         // service name wasn't provided
-        LOG.warn("Message was dropped due to missing service name - '{}'", value);
-        // todo - we may consider adding a logging BurstFilter so that a bad actor cannot
```

Comment:
Please leave the TODO in since it's still valuable.

Reply:
Done.